cobbler/modules/managers/in_tftpd.py

```diff
@@ -107,35 +107,18 @@ def write_boot_files(self):
 
         return 0
 
-    def update_netboot(self, name):
-        """
-        Write out new ``pxelinux.cfg`` files to ``/tftpboot``
-
-        :param name: The name of the system to update.
-        """
-        system = self.systems.find(name=name)
-        if system is None:
-            utils.die("error in system lookup for %s" % name)
-        all_menus = self.tftpgen.get_menu_items()
-        if 'pxe' in all_menus:
-            menu_items = all_menus['pxe']
-            self.tftpgen.write_all_system_files(system, menu_items)
-            # generate any templates listed in the system
-            self.tftpgen.write_templates(system)
-
-    def add_single_system(self, system):
+    def sync_single_system(self, system, menu_items=None):
         """
         Write out new ``pxelinux.cfg`` files to ``/tftpboot``
+        (or grub/system/<mac> in grub case)
```

Comment:
Don't be shy, inline it. A line has more then 80 character... ;) If you look at the config we use 120 everwhere.